lib/vyasa_web/components/contexts/read.ex

```diff
@@ -292,6 +292,7 @@ defmodule VyasaWeb.Context.Read do
     {:noreply,
      socket
      |> assign(:marks, marks |> Mark.edit_mark_in_marks(id, %{body: body}))
+     |> mutate_draft_reflector()
```

Comment:
this follows the same pattern as createMark but not sure why the persistence no work. 💀